python/GafferUI/NodeSetEditor.py

```diff
@@ -377,8 +377,24 @@ def _titleFormat( self, _prefix = None, _maxNodes = 2, _reverseNodes = False, _e
 			if _ellipsis and len( self.__nodeSet ) > _maxNodes :
 				result.append( "..." )
 
+			result.append( "]" )
+
 		return result
 
+	def __updateTitle( self ) :
```

Comment:
I found this choice of names confusing, in that the function didn't actually update the title. Would `__dirtyTitle()` be more a closer description of what is happening here?

Reply:
Good idea, thanks. Fixed in https://github.com/GafferHQ/gaffer/pull/3392/commits/7941677175a07fb7ae2248b651b66f00091bee74